pages/docs/data-ops/managing-resources-with-code.mdx

```diff
@@ -48,6 +87,13 @@ the file structure of your project.
 
 To protect specific resources to only be editable with the Hashboard deployments, add the `preventUpdatesFromUI` flag set as `true` to any config. This prevents users from unintentionally updating critical resources from the Hashboard web application and gives your team greater control over valuable long lived resources that are core to your business needs.
 
-### Syncing resources
+## Builds vs. Project drafts
+
+Builds and [Project Drafts](/docs/project-management/drafts) serve similar purposes -- they both can be used to stage, test, and apply changes. Here is how they differ:
 
-To sync your local config files with edits made to resources via the UI, use the [`pull`](/docs/data-ops/cli/pull) command.
+|                                            | Builds                                          | Project drafts   |
+| ------------------------------------------ | ----------------------------------------------- | ---------------- |
+| Created from...                            | The CLI or git integration                      | The Hashboard UI |
+| Can be edited?                             | No -- the changes within a build are immutable. | Yes              |
+| Supports environment variable substitution | Yes                                             | No               |
+| Can be deployed to multiple projects       | Yes                                             | No               |
```

Comment:
Hmm idk if I agree with the last statement, the resource files can be deployed to multiple projects but the build itself is still tied to a single project just like a draft

Reply:
Intention here is that you can deploy the same yml files to multiple projects for templating / external data sharing. Will reword.